File: src/main/java/com/procurement/notice/NoticeApplication.java
package com.procurement.notice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication
@EnableEurekaClient
public class NoticeApplication {

    public static void main(final String[] args) {
        SpringApplication.run(NoticeApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/notice/NoticeApplication.kt
package com.procurement.notice

import com.procurement.notice.config.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
class NoticeApplication

fun main(args: Array<String>) {
    runApplication<NoticeApplication>(*args)
}
